include/multipass/cli/alias_dict.h

```diff
@@ -21,11 +21,7 @@
 #include <multipass/cli/alias_definition.h>
 #include <multipass/optional.h>
 
-#include <string>
```

Comment:
I don't think it is necessary to remove this as `std::string` is used in the header.

Reply:
Right. `<string>` is nevertheless used in `alias_definition.h` but it doesn't harm to have it here. Added it again.